modules/ROOT/partials/configuration/advanced-typography.adoc

```diff
@@ -0,0 +1,250 @@
+[[content_langs]]
+== `+content_langs+`
+
+This option sets which languages are available for applying language-specific typography rules.
+
+A user can then select text and select from the available languages. That language’s language-specific typography rules are then applied to the selected text.
+
+*Type:* `+Object+`
+
+=== Example: Using `content_langs`
+
+[source,js,subs="attributes+"]
+----
+tinymce.init({
+  selector: 'textarea', // change this value according to your html
+  plugins: '{plugincode}',
+  toolbar: '{plugincode} language',
+  language: 'en_US', // This is required to configure the default language used by typography.
+  content_langs: [
+    { title: 'English', code: 'en' },
+    { title: 'Spanish', code: 'es' },
+    { title: 'German', code: 'de' }
+    ],
+});
+----
+
+=== Currently supported languages
+
+[cols="1,1"]
+|===
+|Supported language |Language code 
+
+|Belarusian
+|be
+
+|Bulgarian
+|bg
+
+|Catalan
+|ca
+
+|Czech
+|cs
+
+|Danish
+|da
+
+|Dutch
+|nl
+
+|English (UK)
+|en-GB
+
+|English (US)
+|en-US
+
+|Esperanto
+|eo
+
+|Estonian
+|et
+
+|Finnish
+|fi
+
+|German
+|de
+
+|Greek
+|el
+
+|Hungarian
+|hu
+
+|Irish
+|ga
+
+|Italian
+|it
+
+|Latvian
+|lv
+
+|Norwegian
+|no
+
+|Polish
+|pl
+
+|Romanian
+|ro
+
+|Russian
+|ru
+
+|Servian
+|sr
+
+|Slovak
+|sk
+
+|Slovenian
+|sl
+
+|Spanish
+|es
+
+|Swedish
+|sv
+
+|Turkish
+|tr
+
+|Ukraninan
+|uk
+
+|===
+
+// typography_default_lang
+// @bforte review required
+[[typography_default_lang]]
+== `+typography_default_lang+`
+
+This option gives the user the ability to pre-set a default typography language within the {productname} editor.  By setting up the `typography_default_lang`, each time the user opens the {productname} editor, it will automatically default to the new default language applied.
```

Comment:
If the user has selected another language before — it will persist between the documents, so this is incorrect.

_By setting up the `typography_default_lang`, if the user opens the {productname} editor for the first time, it will automatically default to the new default language applied. If the language was changed by the user, then it will be saved and persistent between the documents._

Reply:
Replaced introductory `typography_default_lang` paragraph with (lightly edited) re-write by @yacodes.